domains/PolongDev.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "PolongDev",
+        "email": "kennethgregorioperez@gmail.com"
+    },
+    "record": {
+        "URL": "https://polongdev.github.io"
```

Comment:
Page redirects too many times. (Also I see you already have a domain with URL record, you can use https://redirect.pizza)